dev/documentation/en/utilities-email.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Email - Technical Documentation - OJS|OMP|OPS
+---
+
+# Email
+
+Emails are sent with an adapted version of Laravel's [Mailables](https://laravel.com/docs/8.x/mail) class. `PKP\mail\Mailable` extends this class to add support for application requirements such as envelope senders, DMARC compliance, and [Email Templates](email-templates).
+
+In its simplest form, you can use the Mailable class to compose and send an email.
+
+```php
+use Illuminate\Support\Facades\Mail;
+use PKP\mail\Mailable;
+
+$mailable = new Mailable();
+$mailable->from('nate@example.com')
+    ->to('vitaliy@example.com')
+    ->subject('New submission')
+    ->body('A new submission is ready for review.');
+
+Mail::send($mailable);
+```
+
+## Mailables
+
+Most emails sent by the application make use of [Email Templates](email-templates) that can be customized for each journal, press or preprint server. To make use of email templates, create a new Mailable for each email.
+
+```php
+namespace PKP\mail\mailables;
+
+use PKP\mail\Mailable;
+
+class SubmissionReceived extends Mailable
+{
+    /** A custom key for the associated email template */
+    public const EMAIL_KEY = 'SUBMISSION_RECEIVED';
+
+    /**
+     * One or more of the GROUP_ constants
+     *
+     * This is used to group email messages together in the UI
+     * where an editor will find and modify the email templates
+     * used in their journal.
+     */
+    protected static array $groupIds = [
+        self::GROUP_SUBMISSION,
+    ];
+
+    public function __construct()
+    {
+        parent::__construct(func_get_args());
+    }
+}
+```
+
+Use the Mailable to compose and send the email.
+
+```php
+use APP\facades\Repo;
+use Illuminate\Support\Facades\Mail;
+use PKP\mail\mailables\SubmissionReceived;
+
+$emailTemplate = Repo::emailTemplates()->getByKey(
+    $context->getId(),
+    SubmissionReceived::EMAIL_KEY
+);
+
+$mailable = new SubmissionReceived($context, $submission);
```

Comment:
In the example above, according to the `SubmissionReceived` constructor's signature, it doesn't expect any arguments. If not overriding the constructor, this syntax would be correct and the explanation of constructor's arguments role appears below.